### DISCUSSION TITLE
School problem

### DISCUSSION THREAD

**Discussion by zziumofficial** [community/community#102768] [GitHub Education]
### Hi everyone,

I have problem with verification. It says that the school name is not visible but it is very very visible. Github don't have a manual support for it and I don't have any other documents, it is school-ID. What can I do with it?

**Comment by Olayiwolaaa** [comment]
- Contact GitHub Support: Even though GitHub doesn't have manual support for verification, you can still try reaching out to their support team explaining your situation in detail. Provide them with clear images of your school ID showing the name prominently.

- Alternate Documentation: If GitHub doesn't accept your school ID, check if they accept other forms of documentation to verify your affiliation with the school. This could include official enrollment letters, transcripts, or any other documents issued by your school that clearly state your name and the school's name.

- Ask Your School: Reach out to your school's administration office or registrar's office and explain your situation. They might be able to provide you with additional documentation or verification methods that GitHub would accept.

- Verify via Email: Some platforms allow verification via official email addresses provided by the institution. Check if GitHub supports verification through school-provided email addresses. If so, use your school email to verify your affiliation.

- Check Verification Guidelines: Review GitHub's verification guidelines thoroughly to ensure you're following all the requirements correctly. There might be specific formatting or other criteria that you need to meet for successful verification.

- Appeal Process: If your initial verification attempts are unsuccessful, see if there's an appeal process in place. Sometimes, platforms provide an option to appeal verification decisions, allowing you to provide additional evidence or explanation.

- Community Forums: Explore GitHub's community forums or support communities to see if other users have encountered similar issues and have found alternative solutions or workarounds.

- Patience and Persistence: Sometimes, verification processes can be stringent, but persistence and patience can pay off. Keep trying different approaches and reaching out to relevant parties for assistance.

**Comment by marcosfa00** [comment]
Hello @zziumofficial
Could you please provide a bit more detail so that the community can better understand your question?
Are you asking for **Github Copilot**?

**Comment by zziumofficial** [comment]
I have problem with student application.

wt., 30 sty 2024, 17:08 użytkownik Marcos Fernández Avendaño <
***@***.***> napisał:

**Reply by marcosfa00** [reply]
I understand.

On that link, you can find how to apply to the [Student application](https://education.github.com/discount_requests/application). I believe you must be in a regular school in the GitHub database to apply.

Have you followed these steps?

**Comment by zziumofficial** [comment]
Yes. In first  message I attached image. It says that school name is not
visible. It is visible (red) and I don't know what to do.

wt., 30 sty 2024, 17:19 użytkownik Marcos Fernández Avendaño <
***@***.***> napisał:

**Reply by marcosfa00** [reply]
Okay, now I understand.
Don't worry this happened to a few people and should be easy to solve.
You have to ask to your current school for a certificate. **Github** allows **.jpg** files, but it can not be a picture taken by your phone for example. must be clear and well scanned.

On the other hand if you already tried this. try to upload a **Screenshot** in which your name, school name or school ID is readable. 
It does not have to be the complete sheet with the screenshot of the details should be enough

**Comment by zziumofficial** [comment]
I tried with only name - declined. Tried with only school name - school
name is not visible. Tried with grades, school schedule, student's data
from register - only id. School doesn't give certificates so I don't know
what to do.

wt., 30 sty 2024, 17:26 użytkownik Marcos Fernández Avendaño <
***@***.***> napisał:

**Reply by marcosfa00** [reply]
yeah, I know is a bit difficult. it must be all the info in one sheet, can't be separate.
Your school should give you a certificate with says you are studying there.

**Comment by zziumofficial** [comment]
We have a lot of free time now, I would wait about 2 months. In free time I
can do a lot of things so how I can verify by that photo from attachment
but with all data? All 4 corners in but it says that school name isn't
visible. Couldn't it be checked manually by someone?

wt., 30 sty 2024, 17:31 użytkownik Marcos Fernández Avendaño <
***@***.***> napisał:

**Reply by marcosfa00** [reply]
I am so sorry to hear that, but the unique way to do this is by a certificate.
Another option is to upload a screenshot of your profile in your school platform, if you work with this.
For example for Moodle your profile of Moodle in where the URL is clear.

Hope this could help you, but there is no other method, because I've been through this :/

**Comment by zziumofficial** [comment]
School outlook with school email?

wt., 30 sty 2024, 17:38 użytkownik Marcos Fernández Avendaño <
***@***.***> napisał:

**Reply by marcosfa00** [reply]
it could work, but im not sure

**Comment by marcosfa00** [comment]
Oh by the way, if you could mark this conversation as answered by me would help me a lot 😀

**Comment by zziumofficial** [comment]
When I choose option 8- screenshot it says that it need to be ID.....

wt., 30 sty 2024, 17:40 użytkownik Marcos Fernández Avendaño <
***@***.***> napisał:

**Reply by marcosfa00** [reply]
if its a **.jpg** file could not be any problem. but would be fine if I have more info or you send any screenshot here so I know what are you talking about

**Comment by zziumofficial** [comment]
So only jpg?

wt., 30 sty 2024, 17:51 użytkownik Marcos Fernández Avendaño <
***@***.***> napisał:

**Reply by marcosfa00** [reply]
yes, sometimes for example .jpeg fails so be careful

**Comment by zziumofficial** [comment]
But it is impossible. I can only take photo, doesn't matter any option I
would choose, I can't attach image and my photos are always in PNG so how I
can do it...

wt., 30 sty 2024, 17:58 użytkownik Marcos Fernández Avendaño <
***@***.***> napisał:

**Reply by marcosfa00** [reply]
there are a wide range of converters on the internet, just type on google png to jpg.

In some cases if you rename the file to .jpg at the end it works

**Comment by zziumofficial** [comment]
I can't take photo and go to GitHub web and attach it. It needs to be
taken in this moment without any modifications, that's the problem. There
is no option to attach, only take.

wt., 30 sty 2024, 18:02 użytkownik Marcos Fernández Avendaño <
***@***.***> napisał:

**Reply by marcosfa00** [reply]
it should be an option if you are doing it with a computer. if not **DO IT WITH A COMPUTER**

**Reply by zziumofficial** [reply]
![image](https://github.com/community/community/assets/79018671/57d6de3f-5d5c-4c15-8f67-08bd5f9fc483)
there is no option

**Reply by marcosfa00** [reply]
Does not allow you to drop a picture there?

**Reply by zziumofficial** [reply]
it doesn't, when I do it, it just open in a new window

**Reply by zziumofficial** [reply]
when I try to share screenshot from organisations microsoft (school and email from school connected with account is visible) it says that from the photo (I choose option 8)
![image](https://github.com/community/community/assets/79018671/bd9a100a-9496-49f5-b08d-b1eacf27924a)

**Comment by marcosfa00** [comment]
Screenshot of school portal). that means an screenshot of the school website you use with your profile

**Reply by zziumofficial** [reply]
school doesn't have accounts on webside, ID wrong - they cannot read a large letters, nothing works and no button to upload, only front camera because back doesn't work on mobile (it is just black for everybody)

**Reply by marcosfa00** [reply]
you have to scan it, if there is **NO PERFECT QUALITY** then Github would not read it.

Can you mark this as answered please

**Reply by zziumofficial** [reply]
but it is not answered
Scan won't change anything because photo of this scan will be worser than normal photo.

**Reply by marcosfa00** [reply]
you have to scann, the paper, im sorry but I can't do more, you have to think a little bit, I just told you have to do it

**Comment by zziumofficial** [comment]
I sent the promotion certificate with blurred data and I got approved XD

**Reply by JadeBetty** [reply]
for real?
i uploaded my receipt and still got rejected
i should upload my certificate

**Comment by zziumofficial** [comment]
Yes, I got accepted in one seconds and I now have access to benefits.

śr., 31 sty 2024, 05:20 użytkownik JadeBetty ***@***.***>
napisał:
